tests/Fixtures/Integration/misc/braces,phpdoc_indent.test

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Integration of fixers: braces,phpdoc_indent.
```

Comment:
it doesn't fail when we just run one fixer, only when we run them both, and its the braces fixer that crashes!